src/sidebar/test/host-config-test.js

```diff
@@ -16,6 +16,7 @@ describe('hostPageConfig', function() {
       appType: 'bookmarklet',
       openSidebar: true,
       requestConfigFromFrame: 'https://embedder.com',
+      configFrameAncestorLevel: 1,
```

Comment:
I think this change is left over from the first iteration of this PR and no longer applicable?